src/sentry/static/sentry/app/components/contextPickerModal.tsx

```diff
@@ -317,7 +324,7 @@ const ContextPickerModalContainer = createReactClass<ContainerProps, ContainerSt
     return (
       <ContextPickerModal
         {...this.props}
-        projects={projects}
+        projects={projects || []}
```

Comment:
@billyvg If projects was falsy, we didn't show the picker. Now I realize there might be an edge case when the user has no projects. Probably need to test out what happens before and after.

Reply:
I added a case if the user has no projects.

I realize now that having projects being falsy happens when there is no project context that happens if there is no selected org. The computed value `shouldShowProjectSelector` still comes out to false because `organization` is undefined in that case.